packages/ember-glimmer/lib/utils/references.js

```diff
@@ -86,17 +87,54 @@ export class RootReference extends ConstReference {
   }
 }
 
+class TwoWayFlushDetectionTag {
```

Comment:
This class is only used when the flags are enabled, so it can probably be flagged